pages/_app.js

```diff
@@ -1,12 +1,36 @@
 import Layout from '../components/Layout'
 import '../styles/globals.css'
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Head from 'next/head';
+import { ThemeProvider } from '@mui/material/styles';
+import CssBaseline from '@mui/material/CssBaseline';
+import { CacheProvider } from '@emotion/react';
+import theme from '../src/theme';
+import createEmotionCache from '../src/createEmotionCache';
+
+// Client-side cache, shared for the whole session of the user in the browser.
+const clientSideEmotionCache = createEmotionCache();
+
+function MyApp(props) {
+  const { Component, emotionCache = clientSideEmotionCache, pageProps } = props;
 
-function MyApp({ Component, pageProps }) {
   return (
-    <Layout>
-      <Component {...pageProps} />
-    </Layout>
-  )
+    <CacheProvider value={emotionCache}>
+      <Head>
+        <meta name="viewport" content="initial-scale=1, width=device-width" />
+      </Head>
```

Comment:
Could you explain why this meta data is used here as a next/Head instead of inside _documents head

Reply:
https://stackoverflow.com/questions/65832820/next-js-viewport-meta-tags-should-not-be-used-in-document-jss-head

I don't have much time to look into it further at the moment. Also, it's used this way in the example from https://github.com/mui/material-ui/tree/master/examples/nextjs.

Reply:
```suggestion
      <Head>
        <meta name="viewport" content="initial-scale=1, width=device-width" />
      </Head>
```